src/pheval/infra/exomiserdb.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: cp936 -*-
+import logging as log
+import os
+from pathlib import Path
+
+import jaydebeapi
+import polars as pl
+from tqdm import tqdm
+
+info_log = log.getLogger("info")
+info_debug = log.getLogger("debug")
+
+
+class DBConnector:
+    def __init__(self, jar, driver, server, database, user, password):
+        self.jar = jar
+        self.driver = driver
+        self.server = server
+        self.database = database
+        self.user = user
+        self.password = password
+        self.dbconn = None
+
+    def create_connection(self):
+        """creates h2 database connection"""
+        return jaydebeapi.connect(
+            self.driver,
+            f"{self.server}{self.database}",
+            [self.user, self.password],
+            self.jar,
+        )
+
+    def __enter__(self):
+        self.dbconn = self.create_connection()
+        return self.dbconn
+
+    def __exit__(self, *other):
+        self.dbconn.close()
+
+
+class DBConnection:
+    connection = None
+
+    def __init__(self, connection):
+        DBConnection.connection = connection
+
+    @classmethod
+    def get_connection(cls):
+        """Creates return new Singleton database connection"""
+        return DBConnection.connection
+
+    def close(self):
+        return self.connection.close()
+
+    @classmethod
+    def get_cursor(cls):
+        connection = cls.get_connection()
+        return connection.cursor()
+
+
+class ExomiserDB:
+    def __init__(self, db_path: Path):
+        try:
+            self.connector = DBConnector(  # noqa
+                jar=os.path.abspath("./lib/h2-1.4.199.jar"),
+                driver="org.h2.Driver",
+                server=f"jdbc:h2:{db_path}",
+                user="sa",
+                password="",
+                database="",
+            )
+        except Exception as e:
+            print("An exception occurred", e)
+
+    def import_from_semsim_file(self, input: Path, subject_prefix: str, object_prefix: str):
```

Comment:
Same again here with the `input` variable, it would be good to rename it to something else